drivers/shared/executor/executor.go

```diff
@@ -309,7 +309,7 @@ func (e *UniversalExecutor) Launch(command *ExecCommand) (*ProcessState, error)
 	e.childCmd.Env = e.commandCfg.Env
 
 	// Start the process
-	if err = e.start(command); err != nil {
+	if err = wrapNetns(e.childCmd.Start, command.NetworkIsolation); err != nil {
```

Comment:
Q: When would this be in effect?  Do we support network namespaces in raw_exec or windows/macOS cases?

Reply:
Yes namespaces do work with raw_exec, but in linux only.